torch/csrc/jit/init.cpp

```diff
@@ -113,6 +114,26 @@ void initJITBindings(PyObject* module) {
           [](std::shared_ptr<Graph>& g) {
             return EliminateCommonSubexpression(g); // overload resolution
           })
+      .def(
```

Comment:
Idea from document was to have quantizer which calls transformPrep() and transform(). These will have py-bindings and below apis will be invoked from within. Do we want to expose all the below steps explicitly [eg - PropagateQuantInfo,  InsertObserverNodes into python]

Reply:
We're currently exposing all passes to python through `torch._C`, and this diff follows the same practice. At the moment it is used for testing, I don't think we claim it to be an official API.